src/openai/cli/_api/audio.py

```diff
@@ -75,7 +75,18 @@ def transcribe(args: CLITranscribeArgs) -> None:
             # but we don't want to validate that here for forwards-compat
             response_format=cast(Any, args.response_format),
         )
-        print_model(model)
+
+        if args.response_format == 'json':
+            print_model(model)
+        elif args.response_format == 'srt':
+            # Handle SRT response format
+            print_model(model.get('srt'))
+        elif args.response_format == 'vtt':
+            # Handle VTT response format
+            print_model(model.get('vtt'))
+        else:
+            raise CLIError(f"Unsupported response format: {args.response_format}")
```

Comment:
Looks like you've got type checking errors.

I think this isn't quite correct as well, I think it should be this instead:
```py
        model = cast(
            "str | Transcription",
            get_client().audio.transcriptions.create(
                file=(args.file, buffer_reader),
                model=args.model,
                language=args.language or NOT_GIVEN,
                temperature=args.temperature or NOT_GIVEN,
                prompt=args.prompt or NOT_GIVEN,
                # casts required because the API is typed for enums
                # but we don't want to validate that here for forwards-compat
                response_format=cast(Any, args.response_format),
            ),
        )
        if isinstance(model, str):
            sys.stdout.write("\n" + model + "\n")
        else:
            print_model(model)
```
And the same applies below as well (replacing `Transcription` with `Translation`)